mubench.reviewsite/src/Controllers/ReviewsController.php

```diff
@@ -113,7 +120,8 @@ public function postReview(Request $request, Response $response, array $args)
         $this->updateOrCreateReview($misuse_id, $reviewer->id, $comment, $hits);
 
         if ($review["origin"] != "") {
-            return $response->withRedirect("{$this->settings['site_base_url']}{$review["origin"]}");
+            $base_url = substr($this->settings['site_base_url'], -1) === '/' ? substr($this->settings['site_base_url'], 0, -1) : $this->settings['site_base_url'];
+            return $response->withRedirect("{$base_url}{$review["origin"]}");
```

Comment:
So what's the root cause of this '/' problem? I don't like that we're accumulating places where we do this string magic. I think the site_base_url is expected to have a trailing /. Hence, we should generate the origin in a way that does not have a leading /. This should make the string magic here unnecessary.